Frends.IMAP.ReadEmail/Frends.IMAP.ReadEmail/IMAP.cs

```diff
@@ -70,8 +76,10 @@ public static List<EmailMessageResult> ReadEmail([PropertyTab] IMAPSettings sett
                         BodyHtml = msg.HtmlBody,
                         From = string.Join(",", msg.From.Select(j => j.ToString())),
                         To = string.Join(",", msg.To.Select(j => j.ToString())),
-                        Cc = string.Join(",", msg.Cc.Select(j => j.ToString()))
+                        Cc = string.Join(",", msg.Cc.Select(j => j.ToString())),
+                        AttachmentSaveDirs = new List<string>()
                     });
+                    attachments.Add(msg.Attachments);
```

Comment:
1. I did not understand why we need to gather all attachments first into own list and only then process them. Any particular reason for that?
2. The structure of the attachment directory: for each email we receive lets create a subfolder with message ID for example (hopefully that will be valid as dir name? If not, let's create some guid or whatever). And for each message we can have both an `string SavedAttachmentsDirectory ` property and a `List<string> SavedAttachmentsPaths`.